how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -1385,6 +1406,46 @@
 			"unfavoriteIcon": "http://localhost:8080/icons/{scheme}/unfavorite.svg",
 			"favoriteCommand": "/fav",
 			"supportedFavoriteTypes": ["app"]
+		},
+		"lifecycleProvider": {
+			"modules": [
+				{
+					"id": "example-notification-service",
+					"icon": "http://localhost:8080/favicon.ico",
+					"title": "example notification service",
+					"description": "example notification service",
+					"enabled": true,
```

Comment:
Need to disable before merge

Reply:
That was enabled for you to run the test :)